.gitignore

```diff
@@ -48,7 +48,7 @@ yarn-error.log*
 *.launch
 *.sublime-workspace
 
-.vscode/
```

Comment:
With this rule, no matter what we include in the vscode folder and which rules we set, no file will be added to the stage